lib/psql.js

```diff
@@ -11,11 +11,11 @@ function * exec (db, query) {
   debug(query)
   let env = Object.assign({}, process.env, {
```

Comment:
I think the old one used command line arguments but somehow I like env vars more (I think the end result would always be the same since flags override env vars anyways)